Gems/Multiplayer/Code/Include/Multiplayer/Components/SimplePlayerSpawnerComponentBus.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+
+namespace Multiplayer
+{
+    //! @class SimplePlayerSpawnerRequests
+    //! @brief The SimplePlayerSpawnerRequest event-bus exposes helper methods regarding network player spawners.
+    //! For example, it's common for game servers want information regarding valid spawn locations in order to reset a player's position after the player dies or the round restarts (respawning).
+    class SimplePlayerSpawnerRequests : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+        
```

Comment:
This is going on a component, but doesn't use Entity Id as the address? What happens when the component is added multiple times?

From reading the description of the component, maybe it should be turned into a level component so that only one can ever be added?

Reply:
Yep! This is a level component

Reply:
Oh whoops, missed that. ty! :)